parsing/dialects/dialect1/config.fmpp

```diff
@@ -53,10 +52,33 @@ data: {
       "org.apache.calcite.sql.SqlColumnAttributeGeneratedMinValue",
       "org.apache.calcite.sql.SqlColumnAttributeGeneratedOption",
       "org.apache.calcite.sql.SqlColumnAttributeGeneratedStartWith",
+      "org.apache.calcite.sql.SqlColumnAttributeUpperCase",
+      "org.apache.calcite.sql.SqlCreate"
```

Comment:
Looks like this line was accidentally added in the merge.

Reply:
Ok, I'll get rid of it